### PR TITLE
Add star-rail notes to cli

### PR DESCRIPTION
**Pull request by ashleney**
Please confirm these are actual names of the star-rail thingies.

### CODE REVIEW

File: genshin/__main__.py [hunk 1]
```diff
@@ -199,6 +201,32 @@ async def genshin_notes(client: genshin.Client, uid: typing.Optional[int]) -> No
...
+        f"{click.style('Echoes of War:', bold=True)} {data.remaining_weekly_discounts}/{data.max_weekly_discounts}"
+    )
+
+    click.echo(f"\n{click.style('Expeditions:', bold=True)} {data.accepted_epedition_num}/{data.total_expedition_num}")
+    for expedition in data.expeditions:
+        if expedition.remaining_time > datetime.timedelta(0):
+            remaining = f"{expedition.remaining_time} remaining"
+            click.echo(f" - {expedition.name} | {remaining}")
+        else:
+            click.echo(f" - {expedition.name} | Finished")
```
Comment (jokelbaf) [inline genshin/__main__.py]: - Reserve**d** TB power
- Echo~s~ of War
- ~Expeditions~ Assignments
- Daily ~missions~ Training

Comment (ashleney) [reply genshin/__main__.py]: I'm pretty sure echoes of war is correct

Comment (jokelbaf) [reply genshin/__main__.py]: HoYoLab has "Echo" so idk

Comment (ashleney) [reply genshin/__main__.py]: aight

**Comment by jokelbaf** [comment]
Um also what about remaining TB Power recovery time? Am I blind or there isn't one in code

**Comment by ashleney** [comment]
genshin didn't have it and it seemed like a pain, I'll add it

**Review by jokelbaf — APPROVED** [review]

**Pull request merged**